README.md

```diff
@@ -1,2 +1,24 @@
 # node-backend
-Node api for dead crumb trail
+Node rest api for dead-crumb-trail webapp 
+
+## Setup
+Once you clone the repo, follow the below process to run/test/build:
+
+1) Pull Dependencies
+    npm install
+
+2) Create database
+    - Install mongo https://docs.mongodb.com/guides/server/install/
+    - Create a db named starter https://www.mongodb.com/basics/create-database
```

Comment:
There's no need to explicitly create any dbs. The current starter doesn't even need any instances. So just installing mongo and running it in the default port is sufficient